### PR TITLE
fix(ui): Move landscape metrics into top section of home page

### LINKED ISSUE #1836: [Enhancement] Add biomedical "zing" to graphics used on Home page
## Sage Design feedback

Current home page graphic:

Image

At a quick first glance, it is not clear that this is a website dedicated to _biomedical_ challenges, at least, not until some text is read.  

Consider adding a DNA helix or some other type of biomedical-related graphics/image to the main graphic.

### LINKED ISSUE #2145: [Story] Show OC DB metrics in the top part of the home page
### What product(s) is this feature for?

OpenChallenges

### Description

This improvement was suggested by Paul Boutros as a replacement to the "generic" search image we currently have. An alternative would have been to customize the image with "zing" as tracked in #1836.

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by vpchung**
Closes #2145

~Sort of , kinda fixes `#1836`~ (as discussed with @tschaffter )

## Changelog
* Replace main illustration with metrics
* Move plot of tracked challenges into the top section for more visibility
* Remove Landscape section altogether

## Preview

**Before**
Screenshot 2023-09-18 at 2 04 42 PM

**After**
Screenshot 2023-09-18 at 2 05 50 PM

**Alternative design**
Screenshot 2023-09-18 at 2 20 28 PM

### CODE REVIEW

**Comment by tschaffter** [comment]
@vpchung What are the differences between the two new designs?

**Comment by tschaffter** [comment]
The unit tests are broken.

File: libs/openchallenges/home/src/lib/home.component.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h4>
     </h4>
   </div>
   <openchallenges-challenge-search/>
```
Comment (rrchai) [inline libs/openchallenges/home/src/lib/home.component.html:15]: `*sageAppShellNoRender` directive is missing.

**Review by rrchai — CHANGES_REQUESTED** [review]

File: libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html [hunk 2]
```diff
@@ -20,23 +20,68 @@ <h1>
       </p>
     </div>
     <div class="col-4">
-      <img src="/openchallenges-assets/images/graphic-search.svg" alt="Search OpenChallenges" />
+      <div class="row text-center clickable" routerLink="/challenge">
```
[the remark this answers is not in the document]
Comment (vpchung) [reply libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html:23]: Thanks! I must've missed this.

File: libs/openchallenges/home/src/lib/statistics-viewer/statistics-viewer.component.scss [hunk 3]
```diff
@@ -1,67 +1,67 @@
...
-  gap: 32px;
-}
-.section-title {
-  text-align: center;
-  margin-bottom: 42px;
-}
-#statistics {
-  background: #fff;
-  padding: 88px 0;
+// @use 'libs/openchallenges/styles/src/lib/constants';
```
[the remark this answers is not in the document]
Comment (vpchung) [reply libs/openchallenges/home/src/lib/statistics-viewer/statistics-viewer.component.scss:3]: Agreed. I didn't want to do it until the new design is agreed upon.

**Comment by tschaffter** [comment]
@vpchung I would keep the title of the plot "Total Challenges Tracked", which is more concise. It's clear from the plot that it's per year. Also the "total" keyword is important so we should keep it.

The font size used for "annotated challenges" and "organizations & hosting societies" seems to small. We should be proudly state these numbers, not whisper them. ;-)

The two images next to the numbers should be bigger too.

**Comment by vpchung** [comment]
Taking @tschaffter recent feedback into account, I created a couple of versions of the landing section:

### Version A
Original proposed change:

Screenshot 2023-09-19 at 1 56 25 PM

### Version B
Remove metric images:

Screenshot 2023-09-19 at 1 57 56 PM

### Version C
Keep magnifying glass image where it was, move metrics to a new row under the description:

Screenshot 2023-09-19 at 2 02 48 PM

### Version D
Remove images from the landing section altogether, keep magnifying glass image by search bar, and move metrics to a new row under the description:

Screenshot 2023-09-19 at 2 03 48 PM

**Comment by tschaffter** [comment]
I have a preference for Version B because it's better structure and less dense than the other versions to me.

**Comment by tschaffter** [comment]
> From our standup discussion, we will move forward with design D.

One improvement would be to move the plot title closer to the plot than to other elements.

@vpchung Could you please run this design through Adam?

**Comment by vpchung** [comment]
## Final (?) version

Screenshot 2023-09-21 at 8 41 58 PM

FYI - @rrchai I added a negative margin-bottom to the plot title (class=`plot-title`) so that there wouldn't be as much space in-between.  I didn't see any overlap with the chart's current features, but if something does arise, that'd be the place to check first.

**Comment by tschaffter** [comment]
The new design is more elegant. Similar to the previous discussion about grouping elements, the logos featuring the two persons could be moved closer to the number they illustrate. Maybe also make the labels below the number larger or bold and wrapped "organization & hosting societies" on two lines?

**Comment by vpchung** [comment]
Adam's thoughts and feedback:

> I get their feedback, but I think you had it right the first time. It's valuable to have that search box near the top of the page, because that's the call to action. If I were messing around with it, trying to combine both ideas, I would try adding a metric that summarizes the graph (e.g. "215 challenges in 2023") and then pushing the full graph beneath the search area, like you have now. So, the layout would be like this:
> image

I'm a fan with what Adam has mocked up.  @tschaffter @rrchai @chepyle @mdsage1 your thoughts?

**Comment by tschaffter** [comment]
> I'm a fan with what Adam has mocked up, maybe replace the third metric with "platforms"?

@vpchung That's a great idea and a good way to quickly introduce the three OC "core concepts".

Given this ticket and our goal to communicate on OC before the end of next week, could you ask Stockard if she could get a rush order for creating a new illustration to be displayed next to the platform number, as well as augmenting the main illustration with "biomedical, research" elements (reopening #1836 1836)?

**Comment by rrchai** [comment]
> @rrchai I added a negative margin-bottom to the plot title (class=plot-title) so that there wouldn't be as much space in-between. I didn't see any overlap with the chart's current features, but if something does arise, that'd be the place to check first.

The title has default padding, but I don't see any issue using negative margin~

> I'm a fan with what Adam has mocked up, maybe replace the third metric with "platforms"?

good idea!

**Comment by vpchung** [comment]
Adam shared the source of the illustrations ([unDraw](https://undraw.co/search)), and I found a couple we could use for platforms:

### Option A
Screenshot 2023-09-25 at 10 33 38 AM


### Option B
Screenshot 2023-09-25 at 10 33 46 AM


### Option C
Screenshot 2023-09-25 at 10 34 09 AM

**Comment by tschaffter** [comment]
@vpchung All the illustrations you have selected could be used to illustrate challenge platforms.

**Comment by vpchung** [comment]
I personally like B the best, but it lacks a "human" in the design, so it doesn't follow the other metric illustrations.  Between A and C, I like C more.

**Comment by vpchung** [comment]
> [!NOTE]  
> Closing this PR now, as the original proposed layout will not be used.  I will open a new one for [our latest iteration](https://github.com/Sage-Bionetworks/sage-monorepo/pull/2136#issuecomment-1731781362).

**Pull request closed without merging**